source/run-command/run-command.rst

```diff
@@ -0,0 +1,185 @@
+
+* See Server Selection's section on `Use of read preferences with commands <https://github.com/mongodb/specifications/blob/master/source/server-selection/server-selection.rst#use-of-read-preferences-with-commands>`_
+
+If the provided Read Preference is NOT primary, the command sent MUST include the ``$readPreference`` global command argument.
+
+* See OP_MSG's section on `Global Command Arguments <https://github.com/mongodb/specifications/blob/master/source/message/OP_MSG.rst#global-command-arguments>`_
+
+Driver Sessions
+"""""""""""""""
+
+A driver's runCommand API MUST provide an optional session option to support explicit sessions and transactions.
+If a session is not provided the driver MUST attach an implicit session if the selected server supports sessions.
+
+Every ClientSession has a corresponding Logical Session ID representing the server side session ID.
+The Logical Session ID MUST be included under ``lsid`` in the command sent to the server without modifying user input.
+
+* See Driver Sessions' section on `Sending the session ID to the server on all commands <https://github.com/mongodb/specifications/blob/master/source/sessions/driver-sessions.rst#sending-the-session-id-to-the-server-on-all-commands>`_
+
+Transactions
+""""""""""""
+
+If RunCommand is used within a Transaction the read preference MUST be sourced from the Transaction's options.
+The command sent to the server MUST include the transaction specific fields, summarized as follows:
+
+* If ``runCommand`` is executing within a transaction:
+
+  * ``autocommit`` - The autocommit flag MUST be set to false.
+  * ``txnNumber`` - MUST be set.
+
+* If ``runCommand`` is the first operation of the transaction:
+
+  * ``startTransaction`` - MUST be set to true.
+  * ``readConcern`` - MUST be set to the Transaction's readConcern setting if it is NOT the default.
+
+* See Transactions' section on `Generic RunCommand helper within a transaction <https://github.com/mongodb/specifications/blob/master/source/transactions/transactions.rst#generic-runcommand-helper-within-a-transaction>`_
+
+ReadConcern and WriteConcern
+""""""""""""""""""""""""""""
+
+A RunCommand API MUST NOT support Read Concern and Write Concern options.
+Additionally, unless within a transaction, the command sent MUST NOT have any Read Concern or Write Concern fields applied that may be inherited from client, database, or collection options.
+
+* See Read Concern's section on `Generic Command Method <https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#generic-command-method>`_
+* See Write Concern's section on `Generic Command Method <https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#generic-command-method-1>`_
+
+Retryability
+""""""""""""
+
+All commands passed to runCommand are considered non-retryable reads.
```

Comment:
Can we just say "non-retryable" here? I don't think we need to enforce that runCommand is only for reads, if the important factor is saying it's _treated_ like a read operation so we can solicit an optional read preference (and doesn't get default routing to a primary like explicit write operations).

Reply:
Yea, this keeps it succinct